src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6362,15 +6365,20 @@ void FillRemainingPredefinedEntries(EnergyPlusData &state)
                                      3);
                 }
 
-                PreDefTableEntry(state, state.dataOutRptPredefined->pdchOaTaBzInfil, Zone(iZone).Name, ZonePreDefRep(iZone).InfilVolTotalStdDen);
-                totalInfilVol += ZonePreDefRep(iZone).InfilVolTotalStdDen * zoneMult;
+                // Infiltration
+                PreDefTableEntry(state,
+                                 state.dataOutRptPredefined->pdchOaTaBzInfil,
+                                 Zone(iZone).Name,
+                                 ZonePreDefRep(iZone).InfilVolTotalStdDen + ZonePreDefRep(iZone).AFNInfilVolTotalStdDen);
```

Comment:
This change is the source of many table diffs in the AFN files. Previously AFN infiltration was omitted from the Infiltration columns in the Outdoor Air Details report. I should probably post an issue about that.